snippets/csharp/language-reference/keywords/await/await2.cs

```diff
@@ -20,11 +20,13 @@ private static async Task<long> GetPageLengthsAsync(string[] args)
       var client = new HttpClient();
       long pageLengths = 0;
 
-      for (int ctr = 1; ctr < args.Length; ctr++) {
-         var uri = new Uri(Uri.EscapeUriString(args[ctr]));
+      foreach (var arg in args)
+      {
+         var uri = new Uri(Uri.EscapeUriString(arg));
```

Comment:
@Mackiovello note that the `for` loop variable is initialized at 1 (`ctr = 1`). That means that the first element of `args` is skipped.

Reply:
Dang, that's sloppy by me. Thanks for noticing. I'll fix it.